src/main/resources/RecruitmentApp/Configuration.xml

```diff
@@ -31,22 +31,44 @@
 #end
 == Users ==
 {{html}}
-    Specify the space from where to get assignee users: 
-    &lt;form method="post" action=""&gt;
-        &lt;input type="text" name="userSpace" value="$userConfigObj.wiki" /&gt;
+  &lt;form method="post" class="xform" action=""&gt;
+    &lt;dl&gt;
+      &lt;dt&gt;
+        &lt;label for="userSpace"&gt;Users page&lt;/label&gt;
+        &lt;span class="xHint"&gt;Page from where to get assignee users&lt;/span&gt;
+      &lt;/dt&gt;
+      &lt;dd&gt;
+        &lt;input type="text" id="userSpace" name="userSpace" value="$userConfigObj.wiki" /&gt;
+      &lt;/dd&gt;
+    &lt;/dl&gt;
+    &lt;p&gt;
+      &lt;span class="buttonwrapper"&gt;

```

Comment:
Not so mention the old button classes. We are using now the Bootstrap classes .btn .btn-primary, etc. not the .buttonwrapper .button.

Reply:
Also you could have placed the class on the p, and not create an additional span.